operatorcourier/flatten.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import os
+import re
+from yaml import safe_load
+import operatorcourier.identify as identify
+
+logger = logging.getLogger(__name__)
+
+
+def get_flattened_files(source_dir: str) -> list:
+    """
+    :param source_dir: the directory containing different versions
+    of operator bundles (CRD, CSV, package) in separate version directories
+    :return: a list of files needed to be copied to the flattened directory
+    """
+
+    # extract package file and version folders from source_dir
+    file_names, folder_names = [], []
+    for item in os.listdir(source_dir):
+        item_path = f'{source_dir}/{item}'
```

Comment:
This is not a safe way how to construct file paths, use `os.path.join` or `Path` object